src/clj/game/cards/ice.clj

```diff
@@ -493,12 +499,14 @@
                                 (trash state side card))}]}
 
    "Datapike"
-   {:subroutines [{:msg "force the Runner to pay 2 [Credits] if able"
+   {:implementation "Encounter effect is manual"
```

Comment:
Just realised this is here as a mistake. Datapike is fully implemented I believe.